libzkbob-rs-node/Cargo.toml

```diff
@@ -11,11 +11,12 @@ exclude = ["index.node"]
 crate-type = ["cdylib"]
 
 [dependencies]
-libzkbob-rs = { version = "0.3.16", features = ["native"] }
-#libzkbob-rs =  { path = "../libzkbob-rs", features = ["native"] }
+#libzkbob-rs = { version = "0.9.0", features = ["native"] }
```

Comment:
Should this line be uncommented?
Btw, why current version of `libzkbob-rs` in `Cargo.toml` is `0.8.0`?

Reply:
Yes, I uncomment it after the libzkbob-rs library will be published. Current it can be used for the local builds only. 0.9.0 was used as a debug intermediate version. I was publish it to get you ability to test libzkbob-rs-node v0.2.0. The libzkbob-rs-node required published libzkbob-rs to build. So the next libzkbob-rs-node will `0.2.1` and libzkbob-rs: `0.9.1`